fibers.cpp

```diff
@@ -38,23 +38,28 @@
 #include "win.h"
 #include "qpc.h"
 #include <functional>
+#include "gcstring.h"
+#include "ostreamstr.h"
 
 //#include "ostreamcon.h"
 //#define LOG(stuff) con() << stuff << endl
 #define LOG(stuff)
 
+static int fiber_count = 0;
 struct Fiber
 	{
 	enum Status { READY, BLOCKED, ENDED, REUSE }; // NOTE: sequence is significant
 	Fiber()
 		{ }
 	explicit Fiber(void* f, void* arg = nullptr)
-		: fiber(f), status(READY), arg_ref(arg)
-		{ }
+		: fiber(f), status(READY), name(""), arg_ref(arg)

```

Comment:
initializing name twice
and gcstring defaults to "" so you don't need to initialize at all